docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+# Can be enhanced with additional compose file 
+# See also https://docs.docker.com/compose/production/#modify-your-compose-file-for-production
+
+version: '3.3'
+
+services:
+  radicale:
+    image: tomsquest/docker-radicale
+    ports:
+      - "5232:5232"
+    restart: always
+    read_only: true
+    volumes:
+      - data-radicale:/data
+
+volumes:
+  data-radicale:
+    driver: local
```

Comment:
Is this required ?

Per the doc:
> Defaults to whatever driver the Docker Engine has been configured to use, which in most cases is local.
> https://docs.docker.com/compose/compose-file/#driver

Reply:
Nope. This is explicite while it is also set implicit.